frontend/src/component/project/ProjectList/ProjectList.tsx

```diff
@@ -66,7 +74,7 @@ export const ProjectList = () => {
                     actions={
                         <>
                             <ConditionallyRender
-                                condition={!isOss && !isSmallScreen}
+                                condition={!isOss() && !isSmallScreen}
```

Comment:
Fun fact: We don't have project search in "larger than small screens" for [some time now](https://github.com/Unleash/unleash/pull/8814/files#diff-6fe5132bf326860735eea0148fe1c40c9cfb06e5836a9e02bc02e35165638bc0R69) and I guess no one noticed 🙈

Reply:
Likewise... but this is JS, and this is still a common way to check for truthy.

But I agree, would be nice if our linter/editor helped us catch this. Maybe it should require us to be explicit for truthy checks for non-boolean types.

I think this is what we're looking for: https://typescript-eslint.io/rules/strict-boolean-expressions/
But it seems like it's not supported in Biome yet.